spec/lib/services/rating_calculators/for_daily_votes_spec.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Services::RatingCalculators::ForDailyVotes do
+  context 'Verification of rating calculations based on daily votes data:' do
+    it 'standard data' do
+      vote = create(:vote)
+      votes_for_this_server = Vote.where(server_id: vote.server.id)
+      result = Services::RatingCalculators::ForDailyVotes.new(votes_for_this_server).calculate
+      expect(result.round(2)).to eq(0.01)
+    end
+
+    it 'change in the number of votes per day' do
+      vote = create(:vote)
+      vote = create(:vote, server: vote.server)
+      votes_for_this_server = Vote.where(server_id: vote.server.id)
+      result = Services::RatingCalculators::ForDailyVotes.new(votes_for_this_server).calculate
+      expect(result.round(2)).to eq(0.02)
+    end
+
+    it 'limiting the maximum' do
+      vote = create(:vote)
+      100.times { vote = create(:vote, server: vote.server) }
```

Comment:
let's do example for more than maximum, because when it's 100, it does nothing, but when it's 150, it translates 150 into 100

Reply:
(the same for all other tests :) )

Reply:
in fact, there are 101 of them, and I thought it would be better to estimate the minimum threshold for exceeding

Reply:
oh, you're right, didn't notice it, sorry
But it might be better for visibility to create bigger "overweight", (or make it more visible - `101.times{}`)